static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -816,7 +818,7 @@ class ProjectPerformance extends AsyncView<Props, State> {
                       collapsible
                       initiallyCollapsed
                     />
-                    <StyledPanelFooter>
+                    <StyledPanelFooter id={projectDetectorSettingsId}>
```

Comment:
I think it makes more sense to put this on the surrounding div because that whole block is about the detector settings and not just the footer. I tried it out and it seems to scroll everything into view

Reply:
Woops, that's a mistake @narsaynorath. Should be at the div right above my bad.